### PR TITLE
fix: ElevatedView does not show up on Skia

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): fixes Material part of #5186

## PR Type

What kind of change does this PR introduce?

- Bugfix
 
## What is the new behavior?

Make ElevatedView show up on Skia (at least without Elevation)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

**Comment by MartinZikmund** [comment]
Thanks @carldebilly for the help, you fixed the whole thing 😀

File: src/Uno.UI.Toolkit/Uno.UI.Toolkit.Skia.csproj [hunk 1]
```diff
@@ -44,6 +44,11 @@
 		</ProjectReference>
 	</ItemGroup>
 
+	<ItemGroup>
+		<Page Remove="Themes\Generic.xaml" />
```
Comment (davidjohnoliver) [inline src/Uno.UI.Toolkit/Uno.UI.Toolkit.Skia.csproj:48]: 👀

**Review by davidjohnoliver — APPROVED** [review]

File: src/Uno.UI.Toolkit/Uno.UI.Toolkit.Skia.csproj (hunk 1 above)
Comment (carldebilly) [reply src/Uno.UI.Toolkit/Uno.UI.Toolkit.Skia.csproj:48]: That's a required pattern to fix a VS/MSBuild bug ;-)

**Review by carldebilly — APPROVED** [review]

File: src/Uno.UI.Toolkit/Uno.UI.Toolkit.Skia.csproj (hunk 1 above)
Comment (MartinZikmund) [reply src/Uno.UI.Toolkit/Uno.UI.Toolkit.Skia.csproj:48]: ImaginationSpongebobSquarepantsGIF

**Pull request merged**